src/admin/BulkUserCreationPage.tsx

```diff
@@ -32,7 +32,13 @@ const BulkUserCreationPage: FC = () => {
     }),
     onSubmit: ({ role, emailsString }) => {
       const emails = getEmails(emailsString);
-      const command: CreateUserCommand[] = emails.map((email) => ({ email, roles: [role] }));
+      const command: CreateUserCommand[] = emails.map((email) => ({
+        authenticationDetails: {
+          method: AuthenticationMethod.MAGIC_LINK,
+          identifier: email,
```

Comment:
I guess we won't need to bulk create Estonian users, right?

Reply:
Perhaps we should mention it in the description or heading?

Reply:
We will actually. I'll change the method.